process-engine/skeleton
# 🐳 Dockerize Server for Speed Test

## What did you change?

I dockerized the process-engine-server

## How can others test the changes?

have the database-docker-container running, and run `bash build_docker.sh` and `bash start_docker.sh` and see how a dockerized process-engine starts.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
